README.md

```diff
@@ -25,7 +25,34 @@ Here's a oneliner on how I installed everything:
 ```$ sudo pacman -Syu && sudo pacman -S i3-gaps feh i3-dmenu-desktop morc_menu i3status wmctrl```
 
 # Configuration
-## Create a new XSession
+If you are using Plasma >= 5.25, you need to create a new service to run i3 the login to the Plasma(X11) session. If you are using Plasma < 5.25, you need to create a new XSession and login to that.
+
+## Creating a new service (Plasma >= 5.25)
+
+Create a new service file called plasma-i3.service in `/usr/lib/systemd/user` directory as su. 
```

Comment:
This file can be created without root privileges in `$HOME/.config/systemd/user/plasma-i3.service` as well.